portfolio/src/main/webapp/script.js

```diff
@@ -308,113 +323,109 @@ fetchReportMarkers() {
-      /*superMath(route.legs[0].steps[0].start_location, markerTry
-       * ,route.legs[0].steps[0].end_location);*/
-
-      var routeIndex = 0;
-      var lessCrimesInRoute = 1000;
-
-      for (var r = 0; r < result.routes.length; r++) {
-        var route = result.routes[r];
-
-        for (var j = 0; j < route.legs[0].steps.length; j++) {
-          var routeArray = new google.maps.Polyline({
-            path: [
-              new google.maps.LatLng(
-                route.legs[0].steps[j].start_location.lat(),
-                route.legs[0].steps[j].start_location.lng()),
-              new google.maps.LatLng(
-                route.legs[0].steps[j].end_location.lat(),
-                route.legs[0].steps[j].end_location.lng())
-            ]
-          });
-          for (var l = 0; l < markers.length; l++) {
-            var myPosition = new google.maps.LatLng(
-              markers[l].getPosition().lat(), markers[l].getPosition().lng());
-
-            if (google.maps.geometry.poly.isLocationOnEdge(
-                myPosition, routeArray, 0.0005)) {
-              console.log('Relocate!');
-              console.log(l);
-              console.log(route.legs[0].steps[j].end_location.lat());
-              console.log(route.legs[0].steps[j].end_location.lng());
-              counter++;
-            }
-          }
-        }
-
-        if (counter < lessCrimesInRoute) {
-          lessCrimesInRoute = counter;
-          routeIndex = r;
-          console.log(r);
-        }
-        counter = 0;
-      }
-
-      console.log(routeIndex);
-
-      new google.maps.DirectionsRenderer({
-        map: map,
-        directions: result,
-        routeIndex: routeIndex
-      });
+function route() {
+ 
+    var address = document.getElementById('d').value;
+    console.log(address);
+ 
+    var address2 = document.getElementById('e').value;
+    console.log(address2);
+ 
+    var geocoder = new google.maps.Geocoder();
+    var geocoder2 = new google.maps.Geocoder();
+ 
+    geocoder.geocode({ address: address }, function(results, status) {
+      if (status === "OK") {
+        console.log(results[0].geometry.location.lat());
+        destLat = results[0].geometry.location.lat();
+        destLng = results[0].geometry.location.lng();
+      } else {
+          alert("Geocode was not successful for the following reason: " + status);
+      } 
+    });
+ 
+    geocoder2.geocode({ address: address2 }, function(results, status) {
+      if (status === "OK") {
+        console.log(results[0].geometry.location.lat());
+        orgLat = results[0].geometry.location.lat();
+        orgLng = results[0].geometry.location.lng();
+      } else {
+          alert("Geocode was not successful for the following reason: " + status);
+      } 
+    });
+ 
+    let directionsService = new google.maps.DirectionsService(); 
+    let directionsDisplay = new google.maps.DirectionsRenderer(); 
+        directionsDisplay.setMap(null);
```

Comment:
incorrect tabbing

Run the js file through https://beautifier.io/ with **indent 2 spaces** (don't indent 4 spaces)